Test/Integration/ImportExistingCategoryTest.php

```diff
@@ -0,0 +1,116 @@
+        $this->categoryRepository = Bootstrap::getObjectManager()->create(CategoryRepository::class);
+        $this->descriptionHelper = Bootstrap::getObjectManager()->create(DescriptionHelper::class);
+
+
+        $this->orderApiClientMock->method('listTranslatedCategoryForHooks')
+            ->willReturn($this->getCategoryImportData());
+
+        $this->apiCategories = $objectManager->getObject(
+            ApiCategories::class,
+            [
+                'orderApiClient' => $this->orderApiClientMock,
+                'categoryCollectionFactory' => $this->categoryCollectionFactory,
+                'categoryRepository' => $this->categoryRepository,
+                'descriptionHelper' => $this->descriptionHelper
+            ]
+        );
+
+        $this->consumer = $objectManager->getObject(
+            Consumer::class,
+            [
+                'categoriesHelper' => $this->apiCategories
+            ]
+        );
+    }
+
+    /**
+     * Feed queue consumer with category event "created" and verify that chanes applied to magento category
+     *
+     * @magentoConfigFixture current_store storekeeper_general/general/storekeeper_shop_language nl
+     * @magentoConfigFixture current_store general/locale/code nl_NL
+     * @magentoDataFixture StoreKeeper_StoreKeeper::Test/Integration/_files/categories.php
+     */
+    public function testProcess()
+    {
+        //Load existing category
+        $category = $this->categoryFactory->create()->loadByAttribute('url_key', $this->getStoreKeeperTestCategorySlug());
+        $categoryNameBefore = $category->getName();
+        $categoryDescriptionBefore = $category->getDescription();
+        $categoryMetaTitleBefore = $category->getMetaTitle();
+        $categoryMetaKeywordsBefore = $category->getMetaKeywords();
+        $categoryMetaDescriptionBefore = $category->getMetaDescription();
+        
+        //Send webhook data to consumer
+        $this->consumer->process(self::WEBHOOK_DATA);
+        
+        //Reload updated category from repository
+        $category = $this->categoryFactory->create()->loadByAttribute('url_key', $this->getStoreKeeperTestCategorySlug());
+        $categoryNameAfter = $category->getName();
+        $categoryDescriptionAfter = $category->getDescription();
+        $categoryMetaTitleAfter = $category->getMetaTitle();
+        $categoryMetaKeywordsAfter = $category->getMetaKeywords();
+        $categoryMetaDescriptionAfter = $category->getMetaDescription();
+
+        //Assert that old info differs from new info
+        $this->assertNotEquals($categoryNameBefore, $categoryNameAfter);
+        $this->assertNotEquals($categoryDescriptionBefore, $categoryDescriptionAfter);
+        $this->assertNotEquals($categoryMetaTitleBefore, $categoryMetaTitleAfter);
+        $this->assertNotEquals($categoryMetaKeywordsBefore, $categoryMetaKeywordsAfter);
+        $this->assertNotEquals($categoryMetaDescriptionBefore, $categoryMetaDescriptionAfter);
```

Comment:
better to assert what it should equal instead of say it different than before, having all values zeroed would also pass the test